src/Change.php

```diff
@@ -27,19 +27,19 @@ private function __construct(
     }
 
     /** @psalm-pure */
-    public static function added(string $description, bool $isBcBreak): self
+    public static function added(string $description, bool $isBcBreak = true): self
```

Comment:
Not a fan of `= true`, but the future iteration would be to drop it overall, and use new named constructors instead.